ops/storage-ddls/cassandra-setup.cql

```diff
@@ -30,30 +30,26 @@ CREATE TABLE sitesettings(
     cogvisionsvctoken text,
     cogtextsvctoken text,
     insertion_time timestamp,
-    PRIMARY KEY (id, sitename)
+    PRIMARY KEY (sitename)
 );
 
-CREATE INDEX ON sitesettings (sitename);
-
 CREATE TABLE streams (
-  pipeline text,
-  streamid uuid,
-  connector text,
+  pipelinekey text,
+  pipelinelabel text,
+  connectorid uuid,
   params frozen<map<text, text>>,
-  PRIMARY KEY ((pipeline), streamid)
+  PRIMARY KEY (pipelinekey, connectorid)
```

Comment:
IIRC, the old values I was using for `connector` where the values like `FacebookPage` that were passed into the `ConnectorConfig` from each of our stream factories in `project-fortis-spark`. So, I just want to confirm we're OK with converting it to a UUID (assuming to allow for multiple entries of the same pipeline).